app/javascript/controllers/keywords_controller.js

```diff
@@ -19,6 +19,8 @@ export default class extends Controller {
 
     this.searchTarget.value = ''
     this.addItemTarget.insertAdjacentHTML('beforeend', content)
+    // Remove error indications when keyword(s) added
+    this.containerTarget.classList.remove('is-invalid')
```

Comment:
I added this after I created the animated gif. Now the keywords field no longer looks like an error when the user enters a new keyword.